### ISSUE TITLE
[Need Review] Bài về Multi-layer Perceptron

### ISSUE THREAD

**Issue by hitogen** [dlapplications/dlapplications.github.io#3]
Nhờ a @gachiemchiep  và mọi người review cho e bài mới trong series Cơ bản với ạ.

**Comment by hitogen** [comment]
Link bài https://dlapplications.github.io/2018-06-15-MLP/

**Comment by gachiemchiep** [comment]
@hitogen a sẽ review vào trong bài bằng chữ đỏ rồi push lên nhé.

**Comment by gachiemchiep** [comment]
@hitogen a review ở b358e0fcb95fe09add0e449dd4db082e673c8259 

Phần 4, e nói về Backpropagation vẫn còn khó hiểu với sơ sài. 
Nên a nghĩ bỏ nó ra làm một bài khác. 
Về cái bài toán giải thích cho MLP, chỉ cần một bài toán nhỏ như 

Input: điểm văn + toán (2 input)
Bias: Ưu tiên điểm vùng ( 1 điểm)
Hidden: 2 units
Output: đậu hoặc rớt  ( 1 output)

Thế là đủ để họ hình dung rôi.

Những tài liệu em tham khảo khi viết bài post vào đây luôn nhé.

**Comment by gachiemchiep** [comment]
Tí quên. Cuối bài nhớ đề tên tác giả nhé. Không sau này không biết bài nào của ai để sửa

**Comment by hitogen** [comment]
Thanks đại ca @gachiemchiep đã comment rất cụ thể.
E sẽ xem lại và sửa

**Comment by hitogen** [comment]
@gachiemchiep  E đã sửa lại một chút với tách phần 4 (về loss function, gradient descent, ...) ra làm bài riêng sẽ viết sau.

**Comment by hitogen** [comment]
Tài liệu tham khảo
https://github.com/Qian-Han/coursera-Deep-Learning-Specialization/blob/master/Neural%20Networks%20and%20Deep%20Learning/Deep-learning-notation.pdf

**Comment by gachiemchiep** [comment]
Post chưa nhỉ. Nếu rồi thì đóng lại nhé. Nếu chưa thì reviews lần nữa

**Comment by ducanh841988** [comment]
OK nhe!

**Comment by hitogen** [comment]
Chưa post ạ. Bài này chắc để tuần sau. E còn chỉnh sửa tí nữa.

**Comment by gachiemchiep** [comment]
@hitogen vậy lúc nào sửa nt để reviews nhé

**Comment by hitogen** [comment]
@gachiemchiep @ducanh841988  review lần cuối hộ e trc khi lên sóng nha các đại ca

**Comment by ducanh841988** [comment]
@hitogen a thay ok roi nhe

**Comment by gachiemchiep** [comment]
@hitogen bỏ phần "bài toán giải thích cho MLP" rồi à.

**Comment by hitogen** [comment]
@gachiemchiep  dạ e để bài sau nói luôn một thể ạ
